src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -69,7 +69,7 @@ Profile::Profile() :
     _acrylicTransparency{ 0.5 },
     _useAcrylic{ false },
     _scrollbarState{ },
-    _closeOnExit{ false },
+    _closeOnExit{ true },

```

Comment:
There is a similar _closeOnExit in TerminalSettings. Per [doc](/microsoft/Terminal/blob/master/doc/cascadia/TerminalSettings-spec.md) I think it is should be changed too.

Reply:
How embarrassing! And you are absolute right. af3c49c for the other file.